app/code/Magento/Indexer/Model/ProcessManager.php

```diff
@@ -114,6 +114,9 @@ private function multiThreadsExecute($userFunctions)
         // phpcs:ignore Magento2.CodeAnalysis.EmptyBlock,Magento2.Functions.DiscouragedFunction
```

Comment:
I believe this ignoring of empty block could be removed?
```suggestion
        // phpcs:ignore Magento2.Functions.DiscouragedFunction
```

Reply:
updated